app/packages/core/src/components/utils/editor/theme.ts

```diff
@@ -0,0 +1,235 @@
+        '&:after': { content: "'\\eb5f'" },
+      },
+
+      '.cm-completionIcon-enum': {
+        '&:after': { content: "'∪'" },
+      },
+
+      '.cm-completionIcon-function, .cm-completionIcon-method': {
+        '&:after': { content: "'\\ea8c'" },
+      },
+
+      '.cm-completionIcon-interface': {
+        '&:after': { content: "'◌'" },
+      },
+
+      '.cm-completionIcon-keyword': {
+        '&:after': { content: "'\\eb62'" },
+      },
+
+      '.cm-completionIcon-namespace': {
+        '&:after': { content: "'▢'" },
+      },
+
+      '.cm-completionIcon-property': {
+        '&:after': { content: "'□'" },
+      },
+
+      '.cm-completionIcon-text': {
+        '&:after': { content: "'\\ea95'" },
+      },
+
+      '.cm-completionIcon-type': {
+        '&:after': { content: "'𝑡'" },
+      },
+
+      '.cm-completionIcon-variable': {
+        '&:after': { content: "'𝑥'" },
+      },
+
+      '.cm-completionInfo.cm-completionInfo-left': {
+        '&:before': {
+          borderColor: 'transparent',
+          borderStyle: 'solid',
+          borderWidth: '10px',
+          content: "' '",
+          height: '0',
+          position: 'absolute',
+          right: '-20px',
+          width: '0',
+        },
+        marginRight: '12px',
+      },
+
+      '.cm-completionInfo.cm-completionInfo-right': {
+        '&:before': {
+          borderColor: 'transparent',
+          borderStyle: 'solid',
+          borderWidth: '10px',
+          content: "' '",
+          height: '0',
+          left: '-20px',
+          position: 'absolute',
+          width: '0',
+        },
+        marginLeft: '12px',
+      },
+
+      '.cm-completionMatchedText': {
+        fontWeight: 'bold',
+        textDecoration: 'none',
+      },
+
+      '.cm-content': {
+        fontSize: 13,
+        lineHeight: '18.687px',
+        paddingBottom: '8.5px',
+        paddingTop: '8.5px',
+      },
+
+      '.cm-diagnostic': {
+        '&.cm-diagnostic-error': {
+          borderLeft: `3px solid ${theme.palette.error.main}`,
+        },
+      },
+
+      '.cm-gutters': {
+        backgroundColor: '#233044',
+      },
+
+      '.cm-matchingBracket': {
+        fontWeight: 'bold',
+        outline: '1px dashed transparent',
+      },
+      '.cm-nonmatchingBracket': {
+        borderColor: theme.palette.error.main,
+      },
+
+      '.cm-placeholder': {
+        fontFamily: 'monospace',
+      },
+
+      '.cm-scroller': {
+        fontFamily: 'monospace',
+        overflow: 'auto',
+      },
+
+      '.cm-selectionMatch': {
+        backgroundColor: '#e6f3ff',
+      },
+
+      '.cm-tooltip': {
+        backgroundColor: theme.palette.background.paper,
+      },
+
+      '.cm-tooltip.cm-completionInfo': {
+        border: 'none',
+        fontFamily: 'monospace',
+        marginTop: '-11px',
+        maxWidth: 'min-content',
+        minWidth: '250px',
+        padding: '10px',
+      },
+
+      '.cm-tooltip.cm-tooltip-autocomplete': {
```

Comment:
The completion box has the same color like "paper", which makes it a little difficult to distinguish it from the  other elements in the page (see screenshot). Could we use a slightly more `darken`ed tone to make it pop out more?

![image](https://user-images.githubusercontent.com/60616628/229787698-9ca5af27-0e9f-40a4-a2eb-b139830b7019.png)

kinda like in grafana:

![image](https://user-images.githubusercontent.com/60616628/229787914-9608e27d-3bb6-4d03-9896-ce662c150300.png)

Reply:
I switched the colors, so that `cm-tooltip` is using the darker variant of paper and the active selection is using paper.